app/models/box.rb

```diff
@@ -30,7 +30,7 @@ def self.entity_scope
   attribute_field :purpose, copy: true
 
   validates_presence_of :purpose
-  validates_inclusion_of :purpose, in: ->(_) { Box.purposes }
+  validates_inclusion_of :purpose, in: ->(_) { Box.purposes }, allow_blank: true
```

Comment:
This is to avoid a duplicate error message when the purpose is missing:

- purpose can't be blank
- purpose is not included in the list